CHANGELOG.md

```diff
@@ -37,8 +37,10 @@ The changes are relative to the previous release, unless the baseline is specifi
   avifGainMapMetadataDouble structs.
 * Add avif(Un)SignedFraction structs and avifDoubleTo(Un)SignedFraction
   utility functions.
-* Turn on the gain map API by default. Rename the compile flag from
-  AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP to AVIF_ENABLE_GAIN_MAP.
+* Turn on the gain map API. Remove the
+  AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP compile flag.
+  Rename AVIF_ENABLE_EXPERIMENTAL_JPEG_GAIN_MAP_CONVERSION to
+  AVIF_ENABLE_JPEG_GAIN_MAP_CONVERSION.
```

Comment:
Delete the second sentence. `AVIF_ENABLE_EXPERIMENTAL_JPEG_GAIN_MAP_CONVERSION` is an internal build flag and C preprocessor macro, so we don't need to document that it was renamed.